pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/relocation/SegmentRelocator.java

```diff
@@ -169,6 +179,10 @@ private void rebalanceTable(String tableNameWithType) {
     rebalanceConfig.setExternalViewCheckIntervalInMs(_externalViewCheckIntervalInMs);
     rebalanceConfig.setExternalViewStabilizationTimeoutInMs(_externalViewStabilizationTimeoutInMs);
     rebalanceConfig.setUpdateTargetTier(TierConfigUtils.shouldRelocateToTiers(tableConfig));
+    //Do not fail the rebalance when the no-downtime contract cannot be achieved
+    rebalanceConfig.setBestEfforts(_bestEffortsRebalance);
+    rebalanceConfig.setReassignInstances(_reassignInstancesDuringRebalance);
+    rebalanceConfig.setBootstrap(_bootstrapServersDuringRebalance);
```

Comment:
missed _minAvailReplicasDuringRebalance? 

also can expose the boostrap and downtime options as well, as that'd skip even more sanity checks to get rebalance done, just in case needed.

Reply:
Yes, added `downtime` that defaults to `false`.